packages/utils/src/lib/reports/generate-stdout-summary.ts

```diff
@@ -73,8 +72,11 @@ function reportToOverviewSection({
   plugins,
 }: ScoredReport): string {
   const table = new CliTable3({
+    /* eslint-disable no-magic-numbers */
+    colWidths: [38, 7, 8],
+    /* eslint-enable no-magic-numbers */
```

Comment:
If the lint problem only spans one line, then `eslint-disable-next-line` is simpler:

```suggestion
    // eslint-disable-next-line no-magic-numbers
    colWidths: [38, 7, 8],
```

Reply:
done